modules/custom/social_language/social_language.module

```diff
@@ -48,3 +50,26 @@ function social_language_social_group_default_route_tabs_alter(&$tabs) {
   // Unset some tabs created by group.
   unset($tabs['content_translation.local_tasks:entity.group.content_translation_overview']);
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function social_language_form_alter(&$form, FormStateInterface $form_state) {
```

Comment:
@nechai I think it makes more sense to move this code to `social_content_translation.module`.
Also, did you investigate why it doesn't work correct by default? 
Are there some drupal.org issues related to that?

Reply:
@ribel I moved the hook to the "social_content_translation" module. 
This works by default in clear Drupal but in OS field "langcode" is hidden in entities form displays (with default value "en").